build/Targets/Imports.targets

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="OutDir">
```

Comment:
If it is globally specified such as on the Command Line via `/p:OutDir=Value` or in the global property collection when building via the MSBuild APIs. This allows us to modify the value further. 

Users need to be aware that setting it overrides the global property, so in our case, we always want to base it off the existing value (that is appending a value onto the end of the existing value).